front/app/containers/Admin/users/UserTableRow.tsx

```diff
@@ -110,8 +110,13 @@ const UserTableRow = ({
     setShowChangeSeatModal(false);
   };
 
+  const isProposalsEnabled = useFeatureFlag({ name: 'initiatives' });
+
   const handleDeleteClick = () => {
-    const deleteMessage = formatMessage(messages.userDeletionConfirmation);
+    const deleteMessage = isProposalsEnabled
+      ? `${formatMessage(messages.userDeletionConfirmation)}\n\n` +
+        `${formatMessage(messages.userDeletionProposalVotes)}`
+      : `${formatMessage(messages.userDeletionConfirmation)}`;
```

Comment:
This seems too verbose. Any better more React-y way to do this?

I know I can do:
```Javascript
let baseDeleteMessage = `${formatMessage(messages.userDeletionConfirmation)}`;
const deleteMessage = (isProposalsEnabled) ? baseDeleteMessage + `$\n\n{formatMessage(messages.userDeletionProposalVotes)}` : baseDeleteMessage
```
... but it seems we only use `const` to declare variables, and never `let` or `var`.

Reply:
Using `let` is completely fine :+1: